extensions/common/context.cc

```diff
@@ -391,6 +395,19 @@ void populateTCPRequestInfo(bool outbound, RequestInfo* request_info,
   request_info->request_protocol = kProtocolTCP;
 }
 
+bool populateGRPCInfo(RequestInfo* request_info) {
```

Comment:
No need to return boolean here? does not seem to be used and it always returns false.

Reply:
It returns true if it parses correctly (SimpleAtoi return bool). It's not used yet, but seems reasonable to have a status return.